ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -2585,3 +2585,23 @@ def run_ssh_cmd(
             assert "Authentication succeeded" not in stderr
             assert "No more authentication methods to try." in stderr
     return (return_code, stdout, stderr)
+
+
+def is_package_installed(host, pkg):
+    result = host.run_command(
+        ['rpm', '-q', pkg], raiseonerr=False
+    )
+    return result.returncode == 0
+
+
+def remove_package(host, pkg, nodeps=False, raiseonerr=True):
+    cmd = ['rpm', '-e', pkg]
+    if nodeps:
+        cmd.append('--nodeps')
+    result = host.run_command(cmd, raiseonerr=raiseonerr)
+    return result
+
+
+def install_package(host, pkg):
```

Comment:
We already have `ipatests/pytest_ipa/integration/tasks.py::install_packages()` and `uninstall_packages()`, why do we need another pair? This one also doesn't support Debian platform, unlike the previous one.

Reply:
I've adapted the existing code so that it handles --nodeps situations including on Debian. thanks!